charter/index.html

```diff
@@ -92,7 +92,8 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
         <p>Any text rendered like <span class="todo">this</span> refers to content that must be updated when the final charter is published at the latest (e.g., adjusting hyperlinks).</p>
       </div>
 
-      <p class="mission">The <strong>mission</strong> of the <a href="">Solid Working Group <i class="todo">(LINK TBD)</i></a> is to standardize the <cite><a href="https://solidproject.org/TR/protocol">Solid Protocol</a></cite> and its use of associated data interoperability and authentication schemes. This effort will culminate in open standards that can be used by developers of servers and applications to continue to build a rich ecosystem that returns control of data back to users.</p>
+      <p class="mission">The <strong>mission</strong> of the <a href="">Solid Working Group <i class="todo">(LINK TBD)</i></a> is to standardize the <cite><a href="https://solidproject.org/TR/protocol">Solid Protocol</a></cite> and its use of associated data interoperability and authentication schemes.
+      The resulting open standards will ensure interoperability and foster development in a decade-old ecosystem of application and server developers, aiming to return control of data back to users.</p>
```

Comment:
No objection to this change but it is on par with what was said - I don't find it to be an improvement.

Emphasis on "aiming" is good. But "the resulting open standards **will ensure**" sounds way too confident about the future or even how that will be measured.

Reply:
I agree about "will ensure". Will replace with "intend to"